python/ray/data/datasource/parquet_datasource.py

```diff
@@ -61,6 +61,8 @@
 RETRY_EXCEPTIONS_FOR_META_FETCH_TASK = ["AWS Error ACCESS_DENIED", "Timeout"]
 # Maximum number of retries for metadata prefetching task due to transient errors.
 RETRY_MAX_ATTEMPTS_FOR_META_FETCH_TASK = 32
+# Maximum interval in seconds between retries for metadata prefetching task.
+RETRY_MAX_INTERVAL_S_FOR_META_FETCH_TASK = 64
```

Comment:
Nit: any reason for choosing the term "interval" over "backoff"? Not a big deal, but consistency is nice

Reply:
backoff sounds good as well.